cmd/instance_update.go

```diff
@@ -97,6 +99,22 @@ func (c *instanceUpdateCmd) cmdRun(cmd *cobra.Command, _ []string) error {
 					err = globalstate.EgoscaleClient.UpdateInstanceReverseDNS(ctx, c.Zone, *instance.ID, c.ReverseDNS)
 				}
 			}
+			var value egoscale3.UUID
+			var op *egoscale3.Operation
+			value, err = egoscale3.ParseUUID(*instance.ID)
+			if err != nil {
+				return
+			}
+			if c.Protection {
+				op, err = globalstate.EgoscaleV3Client.AddInstanceProtection(ctx, value)
+			} else {
+				op, err = globalstate.EgoscaleV3Client.RemoveInstanceProtection(ctx, value)
+			}
+			if err != nil {
+				return
+			}
+			op, err = globalstate.EgoscaleV3Client.Wait(ctx, op, egoscale3.OperationStateSuccess)
```

Comment:
```suggestion
			_, err = globalstate.EgoscaleV3Client.Wait(ctx, op, egoscale3.OperationStateSuccess)
```
Go doesn't allow you to create unused variables. You have to assign to `_` if you would like to discard a returned value.